topologies/prometheus.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: canaries.flanksource.com/v1
+kind: Topology
+metadata:
+  name: prometheus
+spec:
+  icon: prometheus
+  type: Topology
+  schedule: "@every 5m"
+  components:
+    - name: Pods
+      icon: pod
+      selectors:
+        - name: pods
+          labelSelector: 'namespace=monitoring'
+    - name: Prometheus Database
+      icon: prometheus
+      properties:
+      - name: Metrics count
+        headline: true
+        lookup:
+          prometheus:
+            - query: 'count({__name__=~".+"})'
+              host: 'http://prometheus-k8s.monitoring:9090'
+              display:
+                javascript: |
+                  value = ''
+                  if (results.length > 0) {
+                    value = results[0].value
+                  }
+                  value
+      - name: Disk used (GB)
+        headline: true
+        lookup:
+          prometheus:
+            - query: 'prometheus_tsdb_storage_blocks_bytes / (1024 * 1024 * 1024)'
```

Comment:
Any way to get disk. used percentage + mem/cpu usage + %

Reply:
We can get the total disk via PVC in config-item lookup, but not sure how we can link those

There is a task for using sister properties (https://github.com/flanksource/canary-checker/issues/675), so we can have total disk from config lookup, disk used from prometheus lookup and then use that to get the % maybe